Vagrantfile

```diff
@@ -0,0 +1,131 @@
+# -*- mode: ruby -*-
```

Comment:
Add some hints as to where this file comes from, what it does, and how we plan to eliminate in time its cut and pasting to all our projects